docs/Reference/Pantheon-API-Methods.md

```diff
@@ -1151,6 +1350,37 @@ Returns information about the block by hash.
       }
     }
     ```
+    
+    ```bash tab="curl GraphQL request"
+    curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (hash : \"0xb0efed1fc9326fee967cb2d845d4ebe57c5350a0670c8e86f8052dea6f219f92\") {number transactions{hash} timestamp difficulty totalDifficulty gasUsed gasLimit hash noce ommerCount logsBloom mixHash ommerHash extraData stateRoot receiptsRoot transactionCount transactionsRoot}}"}' http://localhost:8547/graphql
```

Comment:
would be nice to have the same request and results here too so we can compare both RPC and GraphQL queries

Reply:
Have raised https://pegasys1.atlassian.net/secure/RapidBoard.jspa?rapidView=38&view=planning&selectedIssue=PAN-2714 to deal with the wider issue of making all requests again a public network and making the same call for JSON-RPC and GraphQL in each case